yarn.lock

```diff
@@ -4598,9 +4598,11 @@ __metadata:
     "@testing-library/dom": ^9.3.1
     "@testing-library/react": ^14.0.0
     "@testing-library/user-event": ^14.4.3
+    "@types/color": ^3.0.6
```

Comment:
I don't see any imports from this. Where is it used?

Reply:
It defines types from imports from `color`. It's implicitly used in theme.ts.

```ts
import Color from 'color';
```